spec/score_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "score"
+
+RSpec.describe Score do
+  # The score should be 0 before the player starts
+  it 'initializes' do
+    score = Score.new
+    expect(score.score_count(0)).to eq(0)
+  end
+
+  context "players score less than 10 in single frame" do
+    it "adds the score 3 to counter which is at 0" do
+      score = Score.new
+      score.score_count(3)
+      expect(score.score_count(3)).to eq (3)
+    end
+    it "first roll knocks down 5, second 3" do
+      score = Score.new
+      score.score_count(5)
+      score.score_count(3)
+      expect(score.score_count(8)).to eq (8)
```

Comment:
could perhaps create a new method with no args to just read the score for testing as this may be a self fulfilling test at the moment